sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -257,10 +263,9 @@ private static FieldType fromTableFieldSchemaType(
       case "DATE":
         return FieldType.logicalType(SqlTypes.DATE);
       case "DATETIME":
-        // TODO[BEAM-10240]: map to the new logical type when ZetaSQL DATETIME type is supported
-        return FieldType.logicalType(
-            new PassThroughLogicalType<Instant>(
-                "SqlTimestampWithLocalTzType", FieldType.STRING, "", FieldType.DATETIME) {});
+        return FieldType.logicalType(SqlTypes.DATETIME);
```

Comment:
It's possible for users of BigQueryIO to hit this code path through `BigQueryIO.readTableRowsWithSchema()` right? This will be a breaking change since the Java type for a DATETIME field changes from Instant to LocalDateTime. I think that's ok since it's experimental, but we should add a note to CHANGES.md

Reply:
Done. Thanks for catching this.